text-icu.cabal

```diff
@@ -46,6 +46,11 @@ license:        BSD3
 license-file:   LICENSE
 build-type:     Simple
 
+flag homebrew
+  Description: Assume homebrew on macOS. Automatically add /usr/local/opt/ and /usr/homebrew/opt/ paths to extra-lib-dirs and include-dirs.
+  Default:     True
+  Manual:      True
```

Comment:
I suppose `default: true` is for backward compatibility.

However, in general one would not need any `extra-lib-dirs`, because they can be communicated by `pkg-config`.
See how CI does it: https://github.com/haskell/text-icu/blob/be8f0e81c8ea69964d381c40db5bc39095a0258e/.github/workflows/cabal-mac-win.yml#L53-L59

Reply:
Not everyone has pkg-config. In HsOpenSSL for example there are 3 flags: `homebrew-openssl`, `macports-openssl`, and `use-pkg-config`. There's also a non-trivial Setup.hs that tries to auto-detect settings. Probably `text-icu` needs the same.

Reply:
Well, I agree that if one does not have `pkg-config` set up, one does not get good errors from cabal why build is failing, and setting it up is a manual process one would not want to have to do...
So, maybe cabal complaining about non-existent library paths is the lesser evil.